src/encoded/renderers.py

```diff
@@ -314,14 +316,81 @@ def canonical_redirect(event):
     raise HTTPMovedPermanently(location=location, detail="Redirected from " + str(request.path_info))
 
 
+# Web browsers send an Accept request header for initial (e.g. non-AJAX) page requests
+# which should contain 'text/html'
+MIME_TYPE_HTML = 'text/html'
+MIME_TYPE_JSON = 'application/json'
+MIME_TYPE_LD_JSON = 'application/ld+json'
+
+# Note: In cgap-portal, MIME_TYPE_JSON is at the head of this list. In fourfront, MIME_TYPE_HTML is.
+# The cgap-portal behavior might be a bug we should look at bringing into alignment. -kmp 29-Jan-2022
+MIME_TYPES_SUPPORTED = [MIME_TYPE_HTML, MIME_TYPE_JSON, MIME_TYPE_LD_JSON]
```

Comment:
This line is intentionally materially different between `cgap-portal` and `fourfront`. When I rewrote this code for `cgap-portal`, as indicated in the comment, I erroneously made the first element of this list be `MIME_TYPE_JSON`. This wasn't quite right, though is mostly harmless, but it represented a slight incompatible change for `cgap-portal` that I did not intend. I think the right thing may be to fix `cgap-portal` in the future, but for now this just highlights the essential difference between the two implementations. A cascade effect of this is that `MIME_TYPE_DEFAULT` on the next line ends up being `MIME_TYPE_HTML` here but `MIME_TYPE_JSON` in `cgap-portal`. That's probably wrong as well.

This keeps `test_should_transform` passing unmodified. However, some accommodations to newly ported unit tests in `test_renderers.py` had to be made to accommodate this shift. I think they're pretty unobjectionable changes, though.